tiling.py

```diff
@@ -422,29 +711,103 @@ def fundamental_domain_transforms(self):
     def center(self):
         return self._center
 
+    @property
+    def diameter(self):
+        return self._diameter
 
-class DihedralKernel(object):
-    def __init__(self, group):
-        self._group = group
+    @property
+    def radius(self):
+        return self._radius
+
+    @property
+    def group(self):
+        return self._group
+
+
+class vLineKernel(object):
```

Comment:
Again, please don't duplicate entire classes, just add the feature to what's already here. I expect this file should not have to change very much.